/**:
  ros__parameters:
    cycle_time_ms: 10
    # Only one of the three control command topics need be specified
    # "raw", "basic" or "high_level"
    control_command: "basic"
    ne_raptor:
      ecu_build_num: 0xFFFF
      front_axle_to_cog: 1.228
      rear_axle_to_cog: 1.5618
      # 470 degrees on steering wheel ~= 30 degrees on tire
      steer_to_tire_ratio: 15.667
      max_steer_angle: 470.0
      acceleration_positive_jerk_limit: 1.0
      deceleration_negative_jerk_limit: 1.0
      pub_period: 20 # recommended value 20-100ms
    state_machine:
      gear_shift_velocity_threshold_mps: 0.5
      acceleration_limits:
        min: -5.0
        max: 15.0
        threshold: 1.0
      front_steer_limits:
        min: -0.533
        max: 0.533
        threshold: 0.3
      time_step_ms: 100
      timeout_acceleration_mps2: 3.0
      state_transition_timeout_ms: 3000
      gear_shift_accel_deadzone_mps2: 0.5
    features: ["headlights", "horn", "wipers"]
